### PR TITLE
feat: added user model and device gem

### PR DESCRIPTION
**Pull request by PiotrekCodes**
Please accept, my first one :D

### CODE REVIEW

**Comment by zynolia** [comment]
Looks good!

**Review by DariBerrie — APPROVED** [review]
This looks great!

File: app/views/devise/sessions/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<h2>Log in</h2>
+
+<%= simple_form_for(resource, as: resource_name, url: session_path(resource_name),
+  data: {turbo: false}) do |f| %>
+  <div class="form-inputs">
+    <%= f.input :email,
+                required: false,
```
Comment (AmandaPerino) [inline app/views/devise/sessions/new.html.erb:7]: blaaah

Comment (PiotrekCodes) [reply app/views/devise/sessions/new.html.erb:7]: no

**Review by AmandaPerino — CHANGES_REQUESTED** [review]
looks good

**Pull request merged**